rules/java-web-servlet-filter/src/main/resources/otarules.btm

```diff
@@ -20,4 +20,22 @@ DO
   setState($0.getServletContext(), 1);
 ENDRULE
 
+RULE Tomcat servlet filter installation
```

Comment:
nit: `otrarules.btm` in other rule-modules, the name is based on technology/framework.